app/models/twilio_notifier.rb

```diff
@@ -14,7 +14,8 @@ def notify
     client.messages.create(
       from: '+19495372518',
       to: '+12026817334',
-      body: "Thank you for placing an order at Clamorous. Your #{ordered_item_titles} are being prepared by our expert chefs and will be delivered shortly. Your total is $#{@order.order_total_cost.round(2)}, our delivery team accepts all major credit cards."
+      body: "Thank you for placing an order at Droply. Your supplies #{ordered_item_titles} will delivered to #{@order.user.shipping_address}.
+      The total cost of your order was $#{@order.order_total_cost.round(2)}"
```

Comment:
This is fine as is, but why are we having to call `round` on the `@order`? Are you not using BigDecimal to create the `order_total_cost` values?